docs/install.md

```diff
@@ -0,0 +1,30 @@
+# Installation
+
+## Prerequisites
+
+You need Python 3.9 or above in your working environment to use Ragna.
+
+While you can get started with Demo Assistants (Assistants are LLMs), to use Ragna for
+real-world workflows, you
```

Comment:
Should we make this more clear in a separate section? I mean, put the VectorDB <-> SourceStorage and LLM <-> Assistant somewhere at the beginning of the documentation and afterwards just go with SourceStorage and Assistant afterwards?

Reply:
Yes, will make a note for the next PR.